client/src/utils/api-calls/auth.js

```diff
@@ -12,12 +11,17 @@ export const loginUser = async (username, password, dispatch) => {
           }
         );
         if (response.status === 200) {
-            dispatch({type:"LOGIN", payload:{userId:response.data.user._id}})
+            
+            console.log(response);
```

Comment:
Remove console.log after you have tested the API response

Reply:
Removed it.